sanic/request.py

```diff
@@ -94,6 +94,7 @@ class Request(dict):
         "parsed_json",
         "parsed_forwarded",
         "raw_url",
+        "storage",
```

Comment:
maybe `user_storage`? I really want to hammer home that this is not used by sanic and is for the developer

Reply:
@abuckenheimer how about using `custom_context` instead of `storage` or `user_storage` ?